commit 1341cf3d9f51ecefbddb9788f80d21c7b94217ac
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Fri May 30 13:48:42 2014 +0200

    Add rules for naming files in /etc/yum/vars to yum.conf man page.

diff --git a/docs/yum.conf.5 b/docs/yum.conf.5
index c39544d..1596166 100644
--- a/docs/yum.conf.5
+++ b/docs/yum.conf.5
@@ -1225,8 +1225,10 @@ the same name. If the shell environment variable does not exist then the
 configuration file variable will not be replaced.
 
 .LP
-As of 3.2.28, any file in /etc/yum/vars is turned into a variable named after
-the filename (or overrides any of the above variables).
+As of 3.2.28, any properly named file in /etc/yum/vars is turned into
+a variable named after the filename (or overrides any of the above variables).
+Filenames may contain only alphanumeric characters and underscores
+and be in lowercase.
 
 Note that no warnings/errors are given if the files are unreadable, so creating
 files that only root can read may be confusing for users.
